### PR TITLE
[WFLY-9707] Remove the JASPI module from the feature pack as this is now in the core feature pack.

### PR DESCRIPTION
**Pull request by darranl**
:warning: Do not Rebase this PR, the commits in this PR are contained in further topic branches. :warning: 

This pull request depends on the WildFly Core PR https://github.com/wildfly/wildfly-core/pull/3407

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:
- [ x ] Pull Request title is properly formatted: `[WFLY-XYZ] Subject` or `WFLY-XYZ Subject`
- [ x ] Pull Request contains link to the JIRA issue(s)
- [ x ] Pull Request contains description of the issue(s)

The JASPI module is required in WildFly Core for the Elytron subsystem to depend on it, this change is to remove it from WildFly as we do not need to define it twice.

- [ x ] Pull Request does not include fixes for issues other than the main ticket
- [ x ] Attached commits represent units of work and are properly formatted

For bigger changes, major and minor component upgrades make sure your PR also meets following requirements:
- [ ] Pull Request requires a change to the documentation
- [ ] Documentation have been updated accordingly
- [ ] Tests were added to cover changes

For new features ensure as well:
- [ ] Analysis was done
- [ ] Test Plan has been done
- [ ] Tests were verified in advance

If you are not an active contributor of the WildFly project you can request sponsorship by one of the members to help guide you through the process.

### CODE REVIEW

File: servlet-feature-pack/pom.xml [hunk 1]
```diff
@@ -226,11 +226,6 @@
             <artifactId>jboss-el-api_3.0_spec</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>org.jboss.spec.javax.security.auth.message</groupId>
-            <artifactId>jboss-jaspi-api_1.1_spec</artifactId>
-        </dependency>
```
[the remark this answers is not in the document]
Comment (kabir) [reply servlet-feature-pack/pom.xml]: Note to self this has now been done

**Comment by kabir** [comment]
Retest this please

**Comment by darranl** [comment]
@kabir I think we are still missing the related core update for this one.

**Comment by jamezp** [comment]
retest this please

**Pull request merged**